api/endpoints/reservation.md

```diff
@@ -57,6 +57,10 @@ Collection of all Reservations.
     + includeNonCancelled = `true` (optional, boolean, , `false`) ... If true, query will include non-cancelled reservations
     + includePastReservations = `false` (optional, boolean, `true`) ... If true, query will include past reservations
     + showOnlyMyReservations = `true` (optional, boolean, `false`) ... If true, query will only include logged in user's reservations
+    + startDate (optional, number, `1404410211910`) ... Epoch time of reservation start date
+    + endDate (optional, number, `1404421051661`) ... Epoch time of reservation end date
+    + buildingId (optional, number, `44`) ... Id of the building to query from
+    + roomId (optional, number, `16`) ... Id of the room to query from
```

Comment:
Can you double check these? I think these are `building`/`room`, not `buildingId`/`roomId`.